### ISSUE TITLE
Any plans to move to Angular 2

### ISSUE THREAD

**Issue by cloflin** [ISBX/isbx-loopback-cms#54]
Wondering if there are any plans/interest in developing an Angular 2 version?

**Comment by coolwr** [comment]
Yes, we're in the planning stages right now. The goal is to take our learnings up this point and re-architect a new version of the CMS based on Angular 2.
